### ISSUE TITLE
Update to work with most recent version 3.0.0

### ISSUE THREAD

**Issue by roele** [vh/docker-modx#23]
It seems the repository was not updated for a while and the scripts do not work with most recent version (and on a Mac M1). The files should be updated to work with recently released version 3.0.0 and Mac M1 ARM architecture.

**Comment by Aquaengraved** [comment]
@roele Does this container set-up work for 3.0 yet? 
Does it even need to be updated in the first place? Was it tested?

**Comment by roele** [comment]
@Aquaengraved Yes it works, I used it to test an upgrade from 2.8.4 to 3.0.1 on my ARM M1 Mac, hence the mysql:8.0.28-oracle version. An update is required in regards of PHP version and if you want to run it on a M1 Mac (only Oracle seems to provide an ARM image).

**Comment by imtiazShakil** [comment]
hello, I created a pull request https://github.com/modxcms/docker-modx/pull/27 with latest modx 3.0.5.
I've tested it and it works :smile: .

Can you have a look at it?
